node/src/components/rpc_server/rpcs/common.rs

```diff
@@ -78,13 +78,7 @@ pub(super) async fn run_query_and_encode<REv: ReactorEventT>(
 #[serde(rename_all = "snake_case")]
 pub enum ErrorData {
     /// The highest contiguous height range (inclusive) of fully available blocks.
-    HighestContiguousBlockHeightRange((u64, u64)),
-}
-
-impl ErrorData {
-    pub(super) fn highest_contiguous_block_height_range(low: u64, high: u64) -> Self {
-        ErrorData::HighestContiguousBlockHeightRange((low, high))
-    }
+    HighestContiguousBlockHeightRange(ContiguousBlockRange),
```

Comment:
Since we now have a single range, it no longer needs to be called `Highest` (also in the comment).

Reply:
As discussed, I'll rename to `AvailableBlockRange`.

Reply:
Updated in [b834b44](https://github.com/casper-network/casper-node/pull/2789/commits/b834b4484c1125de754a4aac4315b64aca9024d4).